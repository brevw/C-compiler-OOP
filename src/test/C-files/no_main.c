// this c-file does not have a main
int i = 0;
