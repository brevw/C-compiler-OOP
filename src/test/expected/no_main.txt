INT
IDENTIFIER(i)
ASSIGN
INT_LITERAL(0)
SC
Lexing: pass
